architecture/2-UserStories/2.1-QuerySatelliteHealthDataPacket.yml

```diff
@@ -0,0 +1,8 @@
+id: 2.1
+name: Query Satellite Health Data Packet
+actor: Ground Station Operator 
+behavior: Send satellite health data packet downlink command
+rationale: Request satellite health data packet
+derivedFrom:
+- "2-ViewSatelliteHealthDataPacket.yml"
```

Comment:
should be prepended w/ `2-UserStories/`, i.e., `2-UserStories/2-ViewSatelliteHealthDataPacket.yml`